### ISSUE TITLE
Reporting separator role does not have required aria-valuenow attribute

### ISSUE THREAD

**Issue by jongund** [opena11y/evaluation-library#6]
URL: [https://relegant.com/]

**Comment by jongund** [comment]
Fixed bug in beta version
